### ISSUE TITLE
Deleting Default Org Throws Error and Focuses Output Panel

### ISSUE THREAD

**Issue by brandbarr** [forcedotcom/salesforcedx-vscode#1328]
### Summary

Deleting the default scratch org throws an error and focuses the output panel. As scratch orgs are designed to be ephemeral, their deletion should not throw an error Additionally, having to switch back to the terminal in order to make the new default org ruins development flow. 

### Steps To Reproduce:

1.  Create a scratch org  and set it to default - `sfdx force:org:create -f [YOUR CONFIG HERE] -a dev-org -s`
1.  Once the scratch org is created, delete it using `sfdx force:org:delete -u dev-org -p`.
1. The delete process should throw an error.

### Expected result

Deleting a default scratch org should be a regular thing and should not throw/focus on output.

### Actual result

Deleting the default scratch org throws error and focuses the output panel.

### Additional information

I believe this was introduced by #1237 

![Output Hijack](https://user-images.githubusercontent.com/7535323/57483787-9058f580-7275-11e9-8630-3f8f52dbcff3.gif)

**VS Code Version**: Version 1.33.1 (1.33.1)

**SFDX CLI Version**: sfdx-cli/7.5.0-e6ca93eee8 darwin-x64 node-v10.15.3

**OS and version**: 10.14.3

**Comment by lcampos** [comment]
Hi @brandbarr , thanks for providing feedback. We introduced this warning as part of PR #1259 because users were constantly trying to run certain commands (e.g. deploy, retrieve, etc) and running into errors because they didn't have a default username/org set.

We can look at removing the focus on the terminal cc @ntotten

**Comment by CristiCanizales** [comment]
Closing this as the default org can be safely deleted with `SFDX: Delete Default Org` command
Starting SFDX: Delete Default Org

```
20:40:53.290 sf org:delete:scratch --no-prompt
 ›   Warning: @salesforce/cli update available from 2.53.6 to 2.54.6.
Successfully marked scratch org test-hustvqrfscln@example.com for deletion.
20:40:56.846 sf org:delete:scratch --no-prompt
 ended with exit code 0
```
